### PR TITLE
Modfy vesting flow in polygon

### PR DESCRIPTION
**Pull request by tracyarciaga**
## Description of Changes

[Link to Jira Ticket](https://halodao.atlassian.net/browse/HDEV-219)

### Developer Checklist:

* [ ] I have followed the guidelines in our Contributing document
* [ ] This PR has a corresponding JIRA ticket
* [ ] My branch conforms with our naming convention i.e. `feature/HDEV-XXX-description`
* [ ] All files have appropriate file headers and documentation
* [ ] I have written new tests for your core changes, as applicable
* [ ] I have successfully ran tests locally

### Reviewers Checklist:
* [ ] Code is readable and understandable; any unclear parts have explanations 
* [ ] UI/UX changes match the corresponding figma/other design resources, if applicable
* [ ] I have successfully ran tests locally

### CODE REVIEW

File: src/components/VestingModal/index.tsx [hunk 1]
```diff
@@ -68,12 +71,30 @@ interface VestingModalProps {
...
+const AlternativeButton = ({ onClick }: AlternativeButtonProps) => {
+  return (
+    <div className="p-8" onClick={onClick}>
+      Let&apos;s Vest!
+    </div>
+  )
+}
+
+const handleAlternativeButtonClick = () => {
+  window.location.href = 'https://google.com'
```
Comment (0xAplki) [inline src/components/VestingModal/index.tsx]: just wanna know why we want to redirect to google.com?

**Review by 0xAplki — COMMENTED** [review]
lgtm. but waiting for the preview and for the blocks to get resolved

File: src/components/VestingModal/index.tsx (hunk 1 above)
Comment (tracyarciaga) [reply src/components/VestingModal/index.tsx]: @0xAplki ahh I'll change that cos I have the pool links now :)

**Comment by chris247474** [comment]
@tracyarciaga looks good! i wasnt able to test the Lets Vest popup but lets do that on develop in the interest of time

for anyone testing, you need to change network to matic

**Review by 0xAplki — APPROVED** [review]
looks good!

**Pull request merged**